node/rpc/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
```

Comment:
This file needs to be added to `.github/dependabot.yml` b/c it won't be detected automatically.